src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,344 @@
+
+Traces in Sentry are segmented into pieces of [**spans**]({%- link _documentation/apm/apm-glossary.md -%}#span) called [**transactions**]({%- link _documentation/apm/apm-glossary.md -%}#transaction). When instrumenting the application with tracing, collected spans are grouped within an encompassing top-level span called the **transaction span**. This notion of a transaction is specific to Sentry.
+
+If you are collecting transactions from more than a single machine, you will likely encounter [**clock skew**]({%- link _documentation/apm/apm-glossary.md -%}#clock-skew).
+
+## Sampling Transactions
+
+**We strongly recommend sampling your transactions.**
+
+When you enable sampling for APM events in Sentry, you choose a percentage of collected transactions to send to Sentry. For example, if you had an endpoint that received 1000 requests per minute, a sampling rate of 0.25 would result in 250 transactions (25%) being sent to Sentry.
+
+Sampling enables you to collect traces on a subset of your traffic and extrapolate to the total volume. Furthermore, enabling sampling allows you to control the volume of data you send to Sentry and lets you better manage your costs. If you don't have a good understanding of what sampling rate to choose, we recommend you start low and gradually increase the sampling rate as you learn more about your traffic patterns and volume.
+
+When you have multiple projects collecting APM data, Sentry utilizes "head-based" sampling to ensure that once a sampling decision has been made, that decision is propagated to each application or project involved in the [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction). If your applications have multiple entry points, you should aim to choose consistent sampling rates. Choosing different sampling rates can bias your results. Sentry does not support "tail-based" sampling at this time.
+
+If you enable APM collection for a large portion of your traffic, you may exceed your organization's [Quotas and Rate Limits]({%- link _documentation/accounts/quotas.md -%}).
+
+## Setting Up Tracing
+
+{% capture __alert_content -%}
+Sentry's Application Performance Monitoring (APM) features are currently in beta. For more details about APM access, feel free to reach out at [https://sentry.io/for/apm/](https://sentry.io/for/apm/).
+
+Supported SDKs for Tracing:
+- JavaScript Browser ≥ 5.13.0
+- Node.js
+- Python version ≥ 0.11.2
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+### Python
+
+To send any traces, set the `traces_sample_rate` to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+```python
+import sentry_sdk
+    
+sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
+```
+
+**Automating Traces**
```

Comment:
```suggestion
**Automatic Instrumentation**
```

@HazAT @rhcarvalho @untitaker wondering if this is the better title?

Reply:
IMHO yes, "Automatic" or "Automated" Instrumentation sounds better